docs/source/matchers/matchers.rst

```diff
@@ -0,0 +1,42 @@
+Matchers
+========
+
+The first step in comparing a ground truth and predicted graph is computing a matching
+between the two graphs. Currently, traccuracy represents a matching between graphs
+as a matching between the nodes of the graphs - edges are considered matched if the
+endpoints of the edge are matched. Traccuracy supports three types of matchings, depending
+on how many nodes in one graph can be matched to one node in the other graph: 
+* many-to-one: many ground truth nodes may be matched to one predicted node
+* one-to-many: one ground truth node may be matched to many predicted nodes
+* one-to-one: every node can be matched to at most one node in the other graph
+
+Many metrics only operate on one-to-one matchings, or support at most one of the "many"
+mappings. Matchers can operate on the segmentation or directly on the point locations.
+Below is a table summarizing the implemented matchers, what types of matchings
+they can produce, and a brief description of behavior and any hyperparameters.
+
+.. list-table:: Implemented Matchers
+    :widths: auto
+    :header-rows: 1
+
+    * - Matcher
+      - Matching Type(s)
+      - Description
+    * - Point Matcher
+      - one-to-one
+      - Given a maximum distance threshold, the matcher will perform hungarian matching on the points in each
```

Comment:
I think? one thing that works is adding explicit `<br />` HTML directive... but you have to manually place it.